Framework/Source/API/Vulkan/VKRenderContext.cpp

```diff
@@ -175,11 +169,11 @@ namespace Falcor
         {
             for (uint32_t i = 0; i < colorTargets; i++)
             {
-                auto& pTexture = pFbo->getColorTexture(i);
+                auto pTexture = pFbo->getColorTexture(i);
                 if (pTexture) pCtx->resourceBarrier(pTexture.get(), Resource::State::RenderTarget);
             }
 
-            auto& pTexture = pFbo->getDepthStencilTexture();
+            auto pTexture = pFbo->getDepthStencilTexture();
```

Comment:
Fixed. getDepthStencilTexture now returns const &. This cannot be done with color attachments because the vector that would be containing them can be empty.